### PR TITLE
Add content/es/docs/tasks/debug-application-cluster

### PR DESCRIPTION
**Pull request by raelga**
Solves the CLA and rebase issues of kubernetes/website#13981

### CODE REVIEW

**Comment by raelga** [comment]
Muchísimas ganas @emedina!

/approve

File: content/es/docs/tasks/debug-application-cluster/debug-init-containers.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Deberías estar familizarizado con el concepto de [Init Containers](/docs/concepts/abstractions/init-containers/).
+* Deberías conocer la [Configuración de un Init Container](/docs/tasks/configure-pod-container/configure-pod-initialization/#creating-a-pod-that-has-an-init-container/).
+
+{{% /capture %}}
+
+{{% capture steps %}}
```
Comment (sftim) [inline content/es/docs/tasks/debug-application-cluster/debug-init-containers.md:24]: This looks invalid. I'll propose a pull request to fix it.

Comment (raelga) [reply content/es/docs/tasks/debug-application-cluster/debug-init-containers.md:24]: I can do it, this comes from a really old branch. https://github.com/kubernetes/website/pull/13981

Comment (raelga) [reply content/es/docs/tasks/debug-application-cluster/debug-init-containers.md:24]: Done in https://github.com/kubernetes/website/pull/22494

**Pull request merged**